UndertaleModTool/Scripts/Resource Unpackers/ExportSpecificSprites.csx

```diff
@@ -0,0 +1,70 @@
+//Basically a mashup of ExportAllSprites and DumpSpecificCode
+
+using System.Text;
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using UndertaleModLib.Util;
+
+EnsureDataLoaded();
+
+bool padded = ScriptQuestion("Export all sprites WITH padding?");
+
+string texFolder = PromptChooseDirectory();
+if (texFolder == null)
+    throw new ScriptException("The export folder was not set.");
+texFolder = texFolder + "Sprites" + Path.DirectorySeparatorChar;
+Directory.CreateDirectory(texFolder);
+
+TextureWorker worker = new TextureWorker();
+
+List<UndertaleSprite> spritesToDump = new List<UndertaleSprite>();
+List<String> splitStringsList = new List<String>();
+
+string inputtedText = "";
+inputtedText = SimpleTextInput("Menu", "Enter the name of the sprites", "", true);
+string[] individualLineArray = inputtedText.Split('\n', StringSplitOptions.RemoveEmptyEntries);
+foreach (var oneLine in individualLineArray)
+{
+    splitStringsList.Add(oneLine.Trim());
+}
+foreach (string listElement in splitStringsList)
+{
+    foreach (UndertaleSprite spr in Data.Sprites)
+    {
+        if (listElement.ToLower() == spr.Name.Content.ToLower())

```

Comment:
instead of doing converting both tolower and then comparing them, use `listElement.Equals(spr.Name.Content, StringComparison.InvariantCultureIgnoreCase)`